### ISSUE TITLE
When I switch to Esperanto I get TextToSpeech.LANG_NOT_SUPPORTED although it is available

### ISSUE THREAD

**Issue by tema6120** [RHVoice/RHVoice#263]
I am a developer of [ForgetMeNot](https://github.com/tema6120/ForgetMeNot). Having testing different TTS engines in my app I noticed the unusual behavior with Esperanto. When I call
```kotlin
tts.availableLanguages
```
Esperanto appears in the set of languages. If I try:
```kotlin
tts.setLanguage(Locale.forLanguageTag("eo-ZZ"))
```
I get TextToSpeech.LANG_NOT_SUPPORTED. There are no problems with other languages.

**Comment by Olga-Yakovleva** [comment]
Yes, I think this must be a bug I introduced by fixing another one.
Esperanto is a special case, because it's not a language of a country,
so there are no real country codes for it. Android didn't like a
language without a country code, but I wasn't careful enough in my
workaround. I'll be looking into this again.
